app/services/event_log/move_runner.rb

```diff
@@ -16,6 +16,7 @@ def call
         when Event::APPROVE
           move.status = Move::MOVE_STATUS_REQUESTED
           move.date = event.date
+          Allocations::CreateInNomis.call(move) if event.create_in_nomis?
```

Comment:
```suggestion
          Allocations::CreateInNomis.call(move) if event.create_in_nomis? && move&.person&.latest_nomis_booking_id.present?
```

Reply:
I'd prefer to add that check inside the service. Will update.